install_config/aggregate_logging.adoc

```diff
@@ -552,8 +552,8 @@ when `openshift_logging_use_ops` is set to `true`.
 when `openshift_logging_use_ops` is set to `true`.
 
 |`openshift_logging_es_ops_recover_after_time`
```

Comment:
The variable doesn't work. https://bugzilla.redhat.com/show_bug.cgi?id=1822093

Reply:
@jcantrill  Will we still fix the low bug in 3.11?

Reply:
I dont see us fixing this unless there is a customer case that blocks this.  "Ops" deployments are uncommon to begin with

Reply:
Missed this discussion when I was looking at the comments on the bugs themselves earlier. Should the doc change be posted as proposed above then?
@vikram-redhat FYI I'm not sure how to proceed here.

Reply:
@jeana-redhat we should remove this variable and its explanation since it doesn't work. Normally, I wouldn't advocate for this and request a separate bug but in this case the change about time units applied to this variable as well.